polaris.shopify.com/package.json

```diff
@@ -84,15 +85,16 @@
     "generact": "^0.4.0",
     "get-site-urls": "3.0.0-alpha.1",
     "globby": "^11.1.0",
+    "inter-ui": "3.19.3",
```

Comment:
The `satori` package used to generate open graph images doesn't support woff2 or variable width fonts (the one on Shopify's CDN is both of those), so instead we use the version pushed to npm which has a non-variable woff set for us to use.